public/app/listeners/getUserFolder.js

```diff
@@ -0,0 +1,14 @@
+const { GET_USER_FOLDER_CHANNEL } = require('../config/channels');
+const { VAR_FOLDER } = require('../config/config');
+const { ERROR_GENERAL } = require('../config/errors');
+
+const getUserFolder = (mainWindow) => async () => {
+  try {
+    mainWindow.webContents.send(GET_USER_FOLDER_CHANNEL, VAR_FOLDER);
+  } catch (e) {
+    logger.error(e);
```

Comment:
@fboechats, you forgot to import `logger` here. Make sure you run `yarn lint` before committing, as it would have caught this error.